happeo/docs-tools
# Chore/update react styled systems

This package seems to be the last one with react 16 and styled 4.
It is some helpers for docz in the ui kit. Reusable background color and such.

I don't think we want to keep docz around for a long time. But this current fix looks simple enough if it removes our issues with peer dependencies there.

Testing notes:
- run `npm run build` to have updated dist.
- copy paste in the node modules of the ui kit to check that all looks correct
